docs/getting-started/index.md

```diff
@@ -0,0 +1,534 @@
+
+From the `query.proto` we know that response may contain `title` and `body`, so let's modify the last line of the function to return a "Hello!".
+
+```go
+func (k Keeper) Posts(goCtx context.Context, req *types.QueryPostsRequest) (*types.QueryPostsResponse, error) {
+	//...
+	return &types.QueryPostsResponse{Title: "Hello!", Body: "Starport"}, nil
+}
+```
+
+If we start our chain right now and visit the posts endpoint, we would get a "Not Implemented" error. To fix that we need to wire up our API by registering query handlers with gRPC.
+
+Inside `x/chain/module.go` import `"context"`, search for `RegisterGRPCGatewayRoutes` and register query handlers:
+
+```go
+import (
+	//...
+  "context"
+)
+
+func (AppModuleBasic) RegisterGRPCGatewayRoutes(clientCtx client.Context, mux *runtime.ServeMux) {
+	types.RegisterQueryHandlerClient(context.Background(), mux, types.NewQueryClient(clientCtx))
+}
+```
+
+Now we're ready to start our blockchain:
+
+```go
+starport serve
+```
+
+Once the chain has been started, visit [http://localhost:1317/alice/blog/blog/posts](http://localhost:1317/alice/blog/blog/posts) and see our text displayed!
+
+```go
+{
+  "title": "Hello!",
+  "body": "Starport"
+}
+```
+
+The `query` command has also scaffolded `x/blog/client/cli/query_posts.go` that implements a CLI equivalent of the posts query and mounted this command `x/blog/client/cli/query_posts.go` . Run the following command and get the same JSON response:
+
+```go
+blogd q blog posts
+```
+
+## Creating Posts
+
+So far, we've discussed how to modify proto files to define a new API endpoint and modify a keeper query function to return static data back to the user. Of course, a keeper can do more than return a string of data. Its purpose is to manage access to the state of the blockchain.
+
+You can think of the state as being a collection of key-value stores. Each module is responsible for its own store. Changes to the store are triggered by transactions signed and broadcasted by users. Each transaction contains Cosmos SDK messages (not to be confused with proto `message`). When a transaction is processsed, each message gets routed to its module. A module has message handlers that process messages. Processing a message can trigger changes in the state.
+
+### Handling Messages
+
+A Cosmos SDK message contains information that can trigger changes in the state of a blockchain.
+
+To create a message type and its handler, use the `message` command:
+
+```go
+starport message createPost title body
+```
+
+The `message` command accepts message name (`createPost`) and a list of fields (`title` and `body`) as arguments.
+
+The `message` command has created and modified several files:
+
+- modified `proto/chain/tx.proto`
```

Comment:
These files are all in the `blog` instead of `chain` directory in my case

Reply:
Yes, I've replaced `chain` with `blog` everywhere. Thanks!